Breaking change on @module-federation/typescript 2.2.1

After 2.2.1 , node 14 not supported anymore.

Error message: 
```
Error: Cannot find module 'stream/promises'
```

Error come from 

https://github.com/module-federation/universe/blob/main/packages/typescript/src/lib/download.ts#L4

--- a/packages/typescript/src/lib/download.ts
+++ b/packages/typescript/src/lib/download.ts
@@ -1,7 +1,10 @@
 import fs from 'fs';
 import fetch from 'node-fetch';
 import path from 'path';
-import { pipeline } from 'stream/promises';
+import util from 'util';
+import { pipeline } from 'stream';
+
+const pipelineAsync = util.promisify(pipeline);
 
 export interface DownloadOptions {
   /**
@@ -43,7 +46,7 @@ export default async function download(options: DownloadOptions) {
     recursive: true,
   });
 
-  await pipeline(
+  await pipelineAsync(
     response.body,
     fs.createWriteStream(fileDest, {
       flags: 'w',

